lib/jdl/jdl_importer.js

```diff
@@ -184,7 +184,7 @@ function importOneApplicationAndEntities(jdlObject, configuration) {
     exportedDeployments: []
   };
   const jdlApplication = jdlObject.applications[Object.keys(jdlObject.applications)[0]];
-  importState.exportedApplications.push(JHipsterApplicationExporter.exportApplication(jdlApplication));
+  importState.exportedApplications.push(JHipsterApplicationExporter.exportApplication(jdlApplication, configuration));
```

Comment:
this needs testing too

Reply:
I couldn't think test for importOneApplicationAndEntities, exportApplication is duplicated

Reply:
oh okay! I'll see about that, thanks